packages/cli/src/index.ts

```diff
@@ -793,7 +789,7 @@ const handleUnexpected = async (err: Error) => {
 
   // We do not want to render errors about Sentry not being reachable
```

Comment:
@EndangeredMassa We do have this comment here, so I think the debug not rendering errors was intentional.